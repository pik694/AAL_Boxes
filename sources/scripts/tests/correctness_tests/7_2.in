7
91 74 28
70 53 64
12 3 45
85 36 77
74 20 25
3 69 9
55 26 94

