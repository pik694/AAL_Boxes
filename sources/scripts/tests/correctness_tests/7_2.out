2 4 6 5 0 3 1 
836515->796032
